torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -94,6 +94,18 @@ def dfs(module: nn.Module) -> None:
     return modules
 
 
+def _verify_managed_param(name: str, param: nn.Parameter) -> None:
+    """
+    Verify if the parameter is accepted by fully_shard. The only restriction now
+    is that the parameter cannot be a scalar tensor (param.shape == []).
+    """
+    if len(param.shape) == 0:
+        raise ValueError(
+            "fully_shard doesn't support salar parameters. "
+            f"Change {name} to a 1D tensor with numel equals to 1."
```

Comment:
nit: grammar I think
```suggestion
            f"Change {name} to a 1D tensor with numel equal to 1."
```